### PR TITLE
[Portlets] closing feature #9549

### PR DESCRIPTION
**Pull request by NicolasEYSSERIC**

### CODE REVIEW

File: core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java [hunk 1]
```diff
@@ -23,30 +23,45 @@
...
 import org.silverpeas.core.util.StringUtil;
+import org.silverpeas.core.util.URLUtil;
+import org.silverpeas.core.web.portlets.FormNames;
 
 import javax.portlet.*;
 import java.io.IOException;
 
 public class ComponentInstancePortlet extends GenericPortlet implements FormNames {
 
+  private static String PREF_INSTANCEID = "instanceId";
```
Comment (silveruser) [inline core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java]: MINOR Rename this field "PREF_INSTANCEID" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS3008)

File: core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java [hunk 2]
```diff
@@ -23,30 +23,45 @@
...
+      // get portlet window name to hide it (client side)
+      setPortletWindowName(request);
     }
     include(request, response, "portlet.jsp");
   }
 
+  private void setPortletWindowName(RenderRequest request) {
+    String windowId = request.getWindowID();
+    if (StringUtil.isDefined(windowId)) {
+      String portletWindowName = windowId.substring(windowId.lastIndexOf("|")+1);
```
Comment (silveruser) [inline core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java]: MINOR Put single-quotes around '|' to use the faster "lastIndexOf(char)" method. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS3027)

**Comment by silveruser** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java [hunk 3]
```diff
@@ -23,30 +23,45 @@
...
 import org.silverpeas.core.util.StringUtil;
+import org.silverpeas.core.util.URLUtil;
+import org.silverpeas.core.web.portlets.FormNames;
 
 import javax.portlet.*;
 import java.io.IOException;
 
 public class ComponentInstancePortlet extends GenericPortlet implements FormNames {
 
+  private static String PREFINSTANCEID = "instanceId";
```
Comment (silveruser) [inline core-war/src/main/java/org/silverpeas/web/portlets/ComponentInstancePortlet.java]: MINOR Rename this field "PREFINSTANCEID" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS3008)

**Comment by silveruser** [comment]
SonarQube analysis reported 1 issue
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by SilverYoCha** [comment]
Merged.
Closed manually because of manual git operations performed during integration.

**Pull request closed without merging**